samples/sample13/README.md

```diff
@@ -33,7 +33,8 @@
   
     ```
 ### How to run:
-1. Go to `resource/docker` folder and run the build.sh file. This will generate two databases deployment in mysql namespace.
+1. Download `mysql-connector-java-8.0.11.jar` file and add it to a new folder named `libs` inside sample13 directory.
```

Comment:
Can we make `mysql-connector-java-8.0.11.jar` a hyperlink ?

Reply:
Added the link.